diff --git a/node_modules/@zoom/meetingsdk-react-native/android/build.gradle b/node_modules/@zoom/meetingsdk-react-native/android/build.gradle
index 8138ecd..8d3a7ad 100644
--- a/node_modules/@zoom/meetingsdk-react-native/android/build.gradle
+++ b/node_modules/@zoom/meetingsdk-react-native/android/build.gradle
@@ -5,7 +5,6 @@ buildscript {
   repositories {
     google()
     mavenCentral()
-    jcenter()
   }
 
   dependencies {
@@ -60,7 +59,6 @@ android {
 repositories {
   google()
   mavenCentral()
-  jcenter()
 
   def found = false
   def defaultDir = null
