diff --git a/node_modules/react-native-health/RCTAppleHealthKit/RCTAppleHealthKit+Utils.m b/node_modules/react-native-health/RCTAppleHealthKit/RCTAppleHealthKit+Utils.m
index 307056d..a902fcd 100644
--- a/node_modules/react-native-health/RCTAppleHealthKit/RCTAppleHealthKit+Utils.m
+++ b/node_modules/react-native-health/RCTAppleHealthKit/RCTAppleHealthKit+Utils.m
@@ -261,7 +261,7 @@ NSString * const kMetadataKey = @"metadata";
     NSString *anchorString = [options objectForKey:@"anchor"];
     if (!anchorString.length) return nil;
     NSData* anchorData = [[NSData alloc] initWithBase64EncodedString:anchorString options:0];
-    HKQueryAnchor *anchor = [NSKeyedUnarchiver unarchiveObjectWithData:anchorData];
+    HKQueryAnchor *anchor = [NSKeyedUnarchiver unarchivedObjectOfClass:[HKQueryAnchor class] fromData:anchorData error:nil];
     if(anchor == nil){
         return nil;
     }
diff --git a/node_modules/react-native-health/RCTAppleHealthKit/RCTAppleHealthKit.m b/node_modules/react-native-health/RCTAppleHealthKit/RCTAppleHealthKit.m
index e8ddce1..a8da53e 100644
--- a/node_modules/react-native-health/RCTAppleHealthKit/RCTAppleHealthKit.m
+++ b/node_modules/react-native-health/RCTAppleHealthKit/RCTAppleHealthKit.m
@@ -840,8 +840,6 @@ RCT_EXPORT_METHOD(getClinicalRecords:(NSDictionary *)input callback:(RCTResponse
 
 - (void)emitEventInternal:(NSNotification *)notification {
   if (self.hasListeners) {
-    self.callableJSModules = [RCTAppleHealthKit sharedJsModule];
-    [self.callableJSModules setBridge:self.bridge];
     [self sendEventWithName:notification.name
                    body:notification.userInfo];
   }
